src/app/models/trip.js

```diff
@@ -0,0 +1,71 @@
+// Defining the Model
+
+import Backbone from 'backbone';
+
+const Trip = Backbone.Model.extend({
+
+
+  urlRoot: 'https://ada-backtrek-api.herokuapp.com/trips/',
+
+  validate: function(attributes) {
+    const errors = {};
+    const CONTINENTS = ['Asia', 'Africa', 'Australasia', 'Europe', 'South America', 'North America', 'asia', 'africa', 'australasia', 'europe', 'south america', 'north america']
+    if (!attributes.name) {
+      errors['name'] = [' cannot be blank.'];
+    } //end name attribute
+
+    if (!attributes.category) {
+      errors.category = [' cannot be blank']
+    }
+
+    if (!attributes.continent) {
+      errors.continent = [' cannot be blank'];
+    }
+
+    if (!CONTINENTS.includes(attributes.continent)) {
+      errors.continent = [' is not a valid continent.'];
```

Comment:
This would overwrite the blank continent message.  Instead you'll need to do:

```
  if (errors.continent) {
   errors.continent.push('is not a valid continent');
  } else {
    errors.continent = [' is not a valid continent.'];
  }
```